<?xml version="1.0"?>

<launch>
  
  <arg name="offline" default="false" />
  <param name="use_sim_time" value="$(arg offline)" />


  <!-- Launch Perception node -->
  <node pkg="yolo_ekf" type="yolo_ekf" name="yolo_ekf" output="screen" /> 

  <!-- Launch YOLOv3 -->
  <!-- <include file="$(find darknet_ros)/launch/yolo_v3.launch"> </include> -->
  


</launch>
